src/client/pages/OfferNew/Introduction/Sidebar/DetailsModal/utils.ts

```diff
@@ -340,14 +516,16 @@ export const getInitialSwedishHouseValues = (
       })),
   },
 })
-
+type NorwegianHomeContentsDetailsAlias = NorwegianHomeContentsDetails & {
```

Comment:
The purpose of the alias is that you have to use alias for type  on NorwegianHomeContentsDetails, DanishHomeContentsDetails. since "type" can be different things. (If you try to generate the graphql it will throw an error) So here the mapping from danishHomeType-> type occurs. 
Maybe I can try to explain it a bit better. because There might be a better solution/naming

Reply:
Right! Yeah lets have a chat maybe :)